xml/nm.xml

```diff
@@ -142,6 +142,22 @@ http://www.heise.de/netze/artikel/WLAN-und-LAN-sichern-mit-IEEE-802-1X-und-Radiu
    connections.
   </para>
 
+  <tip>
+   <title>&nm; Connection Editor</title>
+   <para>
+    In previous &productname; releases, network connections were configured
+    using the application called <emphasis>&nm; Connection Editor</emphasis>.
+    It is no longer installed by default because <emphasis>&gnome; Control
+     Center</emphasis> has fully replaced its configuration capabilities.
+   </para>
+   <para>
+    If you still need to use &nm; Connection Editor to configure network
```

Comment:
```suggestion
    If you still need to use the &nm; Connection Editor to configure network
```

Reply:
Question rather than suggestion: would it not be better to say "if you still _want_ to use..." rather than "need to"? If the Control Center really does everything then strictly it is not a need, right?

Reply:
If they refuse to use GNOME and prefer for example XFCE instead, they need to use it. And exactly for such people we're mentioning the connection editor.